src/allmydata/test/test_deepcheck.py

```diff
@@ -945,7 +946,7 @@ def _delete_some_shares(self, node):
     def _corrupt_some_shares(self, node):
         for (shnum, serverid, sharefile) in self.find_uri_shares(node.get_uri()):
             if shnum in (0,1):
-                yield run_cli("debug", "corrupt-share", sharefile)
+                yield run_cli("debug", "corrupt-share", native_str(sharefile))
```

Comment:
This seems correct but also a bit out of place.  This is a tiny PR and this is a tiny hunk so I don't mind if it stays but I'm curious why it ended up here.

Reply:
`sharefile` is now unicode, so on Python 2 this breaks since CLI doesn't like unicode at this point.

Reply:
There's going to have to be a second pass of ripping out all the dual-2-and-3-isms...